experimental/LieAlgebras/src/LieAlgebra.jl

```diff
@@ -57,6 +57,13 @@ function basis(L::LieAlgebra, i::Int)
   return L([(j == i ? one(R) : zero(R)) for j in 1:dim(L)])
 end
 
+@doc raw"""
+    characteristic(L::SimpleLieAlgebra{T}) -> Int64
+
+Return the characteristic of the base ring of the Lie algebra `L`.
+"""
+characteristic(L::LieAlgebra) = characteristic(base_ring(L))
```

Comment:
Just for your information:
Docstring needs to be adjusted, docstring not included in docu, test missing
-> everything already fixed by me